docs/src/mlj_cheatsheet.md

```diff
@@ -65,11 +69,11 @@ type                                       | scitype
 
 Use `schema(X)` to get the column scitypes of a table `X`
 
-`coerce(y, Multiclass)` attempts coercion of all elements of `y` into scitype `Multiclass`
+- `coerce(y, Multiclass)` attempts coercion of all elements of `y` into scitype `Multiclass`
```

Comment:
Should we mention, before this bullet, something like "Some example to coerce given data:"? Just thinking.